MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -527,5 +527,22 @@ async void ValidateAndSaveForm()
     }
 }
 
+{% endhighlight %}
+{% endtabs %}
+
+## Adding custom information to a form field
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to assign custom information to form fields. The `CustomData` property is utilized to store additional information about each form field instance for reference. However, it's essential to note that these data are solely intended for reference purposes and will not be displayed in the PdfViewer interface.
+
+The following code sample demonstrates how to set the `CustomData` property for a form field. In this code snippet, we set the custom data to represent the modification time of the form field.
+
+{% tabs %}
+{% highlight C# %}
+
+private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChangedEventArgs e)
```

Comment:
<img width="382" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/46017c41-c0ff-4b13-af46-2cd6fbcda92c">


We may mention text also to mention before as below. 

**Modified Date is:** {date}

Reply:
Changed.